dcrutil/wif.go

```diff
@@ -153,13 +153,3 @@ func (w *WIF) SerializePubKey() []byte {
 func (w *WIF) DSA() int {
 	return w.ecType
 }
-
-// paddedAppend appends the src byte slice to dst, returning the new slice.
-// If the length of the source is smaller than the passed size, leading zero
-// bytes are appended to the dst slice before appending src.
-func paddedAppend(size uint, dst, src []byte) []byte {
```

Comment:
Rather than removing this, I think it should be compared to btcd upstream.  There is a reason it exists and it was to avoid extra allocations.  It appears the dcrd code was changed to be less efficient and shouldn't have.

Reply:
After looking at this more closely, the code was changed to use an interface, so the optimization used upstream can't be used here and thus it's acceptable to remove it.